discovery-provider/src/queries/get_remixable_tracks.py

```diff
@@ -0,0 +1,61 @@
+from sqlalchemy import desc
+from src.models import Track, Stem
+from src.queries.query_helpers import create_save_repost_count_subquery, \
+    get_current_user_id, populate_track_metadata, add_users_to_tracks, decayed_score
+from src.utils.db_session import get_db_read_replica
+from src.utils import helpers
+
+def get_remixable_tracks(args):
+    """Gets a list of remixable tracks"""
+    current_user_id = get_current_user_id()
```

Comment:
hmm so we're using the non_param get_current_user_id function (the one that checks request header) here which by the way will raise an exception if the header is not passed in cuz default is required=True, and i noticed you're not using the current_user_id you defined in your endpoint args that is the input to this function. Should probably consolidate i.e. use the one in args, or this one and remove it from args based on how we foresee the client making the request?

Lmk if this doesnt make sense

Reply:
not great we have these 2 funcs with the same exact name (get_current_user_id). we should do sthg about this :)